### PR TITLE
Feature: Hosts can have a BACnet Instance Number

### PR DESCRIPTION
**Pull request by oyvindhagberg**
This commit adds subcommands to the host command to assign BACnet IDs to hosts.
A BACnet ID (instance number) is an integer in the range 0-4194302, and hosts can't share the same ID.

### CODE REVIEW

File: mreg_cli/bacnet.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+from .cli import Flag, cli
+from .exceptions import HostNotFoundWarning
+from .history import history
+from .host import host
+from .log import cli_error, cli_info, cli_warning
+from .util import (
+    clean_hostname,
+    host_info_by_name,
+    _host_info_by_name,
+    resolve_input_name,
```
Comment (mbakke) [inline mreg_cli/bacnet.py:10]: Some of these imports are unused.

File: mreg_cli/host.py [hunk 2]
```diff
@@ -459,6 +459,14 @@ def print_txt(txt: str, padding: int = 14) -> None:
     print("{1:<{0}}{2}".format(padding, "TXT:", txt))
 
 
+def print_bacnetid(bacnetid: dict, padding: int = 14) -> None:
+    """Pretty print given txt"""
+    if bacnetid is None:
+        return
+    assert isinstance(bacnetid, dict)
```
Comment (mbakke) [inline mreg_cli/host.py:466]: Is this necessary, given the type annotation?

**Review by mbakke — APPROVED** [review]
Apart from a few minor comments, everything LGTM :shipit:

File: mreg_cli/bacnet.py (hunk 1 above)
Comment (oyvindhagberg) [reply mreg_cli/bacnet.py]: Fixed now in b1ce84c1afd93f8751371d2e2355d02d6c930b7a

**Pull request merged**